selenium/Resources/login.robot

```diff
@@ -0,0 +1,30 @@
+*** Settings ***
+Library           SeleniumLibrary
+Library           OperatingSystem
+Library           String
+
+*** Keywords ***
+Provider Home Page
+    Wait Until Page Contains    PROVIDER HOME    30 sec
+
+Open Browser To Login Page
+    Open Browser    ${LOGIN_URL}    ${BROWSER}
+    Title Should Be    Telemed
+
+Input Username
```

Comment:
Well, this is close but not yet there :) 
A good page abstraction needs to maintain a balanced level of generalization. 
More on Slack